internal/publisher/kafka.go

```diff
@@ -41,10 +62,24 @@ func (p *KafkaPublisher) Close() error {
 	return p.producer.Close()
 }
 
+func (p *KafkaPublisher) keyData(e *Event) []byte {
+	if data, ok := e.Data[p.cfg.MessageKeyFrom]; ok {
+		key, _ := json.Marshal(data)
+		return key
+	}
+
+	return []byte(e.Table)
```

Comment:
I would have returned `nil` here since sarama partitioners switch to a random partitioner if message has a `nil` key, which is generally the behavior one would expect. The drawback of this kind of constant value, is that if for some reason you got your configuration wrong or dropped the column configured from the origin table and forget to update your configuration, you will produce every single message in the same partitions and impact your consumer performance and eventually put some of your broker(s) at risk